eslzArm/subscriptionTemplates/avnmConfiguration.json

```diff
@@ -0,0 +1,690 @@
+                                "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', 'avnm-ng-all')]",
+                                "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', format('avnm-ng-{0}', parameters('location')))]",
+                                "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', format('avnm-ng-{0}', parameters('locationSecondary')))]"
+                            ],
+                            "properties": {
+                                "appliesToGroups": [
+                                    {
+                                        "networkGroupId": "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', 'avnm-ng-all')]"
+                                    },
+                                    {
+                                        "networkGroupId": "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', format('avnm-ng-{0}', parameters('location')))]"
+                                    },
+                                    {
+                                        "networkGroupId": "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', format('avnm-ng-{0}', parameters('locationSecondary')))]"
+                                    }
+
+                                ]
+                            }
+                        },
+                        {
+                            "condition": "[equals(parameters('enableSecondaryRegion'), 'No')]",
+                            "type": "Microsoft.Network/networkManagers/securityAdminConfigurations/ruleCollections",
+                            "apiVersion": "2024-03-01",
+                            "name": "[format('{0}/{1}/{2}', 'avnm', 'sac-alz', 'rc-ALZ')]",
+                            "dependsOn": [
+                                "[resourceId('Microsoft.Network/networkManagers/securityAdminConfigurations', 'avnm', 'sac-alz')]",
+                                "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', 'avnm-ng-all')]",
+                                "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', format('avnm-ng-{0}', parameters('location')))]"
+                            ],
+                            "properties": {
+                                "appliesToGroups": [
+                                    {
+                                        "networkGroupId": "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', 'avnm-ng-all')]"
+                                    },
+                                    {
+                                        "networkGroupId": "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', format('avnm-ng-{0}', parameters('location')))]"
+                                    }
+
+                                ]
+                            }
+                        },
```

Comment:
if my above comment is true, this isnt need also right as you just apply to the `all` one and job done, no matter how many regions